MISC-363 update 2022.13.2 (#111)
gatling/frontline-cloud-doc: content/reference/user/reports/index.md

This bar is a combination of buttons including:

- **Start / Stop**: Start a new run of the simulation, or stop the ongoing run

> - **Generate public link**: Create a [public link]({{< ref "#public-links" >}}) that can be viewed by anyone it is sent to.

Reviewer: ```suggestion
- **Share**: Create a [public link]({{< ref "#public-links" >}}) to share this reports to anyone you want.
```

{{< img src="timeline.png" alt="Timeline" >}}

## Assertions

> The image below shows the status of the simulation (Ongoing, successful, timeout...).

Reviewer 2: Either use capitals or not, but don't mix
